test/sqltest/data_record.sql

```diff
@@ -12,4 +12,6 @@ WHERE
     data_record.form <> couchdb.doc->>'form' OR
     data_record.patient_id <> couchdb.doc->>'patient_id' OR
     data_record.contact_uuid <> couchdb.doc->'contact'->>'_id'
+    OR -- deleted is true
+    (data_record.deleted = true)
```

Comment:
this condition conflicts with (data_record.uuid IS NULL) so that this test will always fail

if it is deleting correctly, there will be some rows in couchdb but not in data_record.
if its not deleting correctly, then there will be some rows where data_record.deleted is true

Reply:
Thanks. I have updated the tests.